build-system/tasks/visual-diff/.percy.yaml

```diff
@@ -0,0 +1,6 @@
+version: 1
```

Comment:
If this file supports comments, maybe add a top-line comment explaining what it's for, and a link to Percy's reference docs for the yaml file format?

Reply:
Done